diff --git a/tf2_py/src/tf2_py.cpp b/tf2_py/src/tf2_py.cpp
index b78ab67..b1b2068 100644
--- a/tf2_py/src/tf2_py.cpp
+++ b/tf2_py/src/tf2_py.cpp
@@ -86,7 +86,7 @@ static PyObject *transform_converter(const geometry_msgs::TransformStamped* tran
     return NULL;
   }
 
-  pinst = PyEval_CallObject(pclass, pargs);
+  pinst = PyObject_CallObject(pclass, pargs);
   Py_DECREF(pclass);
   Py_DECREF(pargs);
   if(pinst == NULL)
